pkg/routes/logs.go

```diff
@@ -43,7 +44,12 @@ func (l Logs) Install(c *restful.Container) {
 func logFileHandler(req *restful.Request, resp *restful.Response) {
 	logdir := "/var/log"
 	actual := path.Join(logdir, req.PathParameter("logpath"))
-
+	//According to NCC-E003660-B4Y. Although http.Server explicitly sanitizes its input,
+	//before using the "actual" path, it should be ensured that it is within the logdir directory.
+	if !strings.HasPrefix(actual, logdir+"/") {
```

Comment:
Don't we currently support an empty `logpath` parameter listing log content?

Should this be:

```go
actual != logdir || !strings.HasPrefix(actual, logdir) || !actual[len(logdir)] == '/')
```

Add a test case to cover this matching exactly

Reply:
@liggitt PTAL.Thank you for your helpful suggestion.